### PR TITLE
Feat/dry

### PR DESCRIPTION
**Pull request by xzdry**
1. add HTML component to show html content    
2. Add content and label slot to Timeline component

### CODE REVIEW

File: packages/runtime/src/components/core/HTML.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    slots: {},
+    styleSlots: ['content'],
+    events: [],
+  },
+})(({ elementRef, customStyle, html }) => {
+  return (
+    <div
+      ref={elementRef}
+      className={cx(customStyle?.content)}
+      dangerouslySetInnerHTML={{ __html: html }}
```
Comment (Yuyz0112) [inline packages/runtime/src/components/core/HTML.tsx:37]: worth a try?
https://github.com/aknuds1/html-to-react

Comment (tanbowensg) [reply packages/runtime/src/components/core/HTML.tsx:37]: Perhaps we'd better move this component to the project, not in runtime.

Comment (xzdry) [reply packages/runtime/src/components/core/HTML.tsx]: The html components have been removed and will be placed in a custom component library

**Review by tanbowensg — APPROVED** [review]
LGTM

**Pull request merged**